zathura/zathura.c

```diff
@@ -651,13 +652,14 @@ document_open(zathura_t* zathura, const char* path, const char* password,
   }
 
   /* read history file */
+  char first_page_column_list_default[] = "1:2";
```

Comment:
Hmm.. True, but it is not really a buffer, but merely a default value. The only case it is modified is when `file_info.page_column_list` is an empty string, in which case the same string "1:2" is written by the statement
```C
girara_setting_get(zathura->ui.session, "first-page-column", &first_page_column_list);
```
However, I must admit that I don't quite understand, where that value comes from. ;-)

What am I missing?